.github/workflows/package-workflow.yml

```diff
@@ -0,0 +1,38 @@
+name: E4E Packaging Workflow
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    env:
+      PROJECT_NAME: explorer-for-endevor
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+        # order operating systems from best to worst
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Get E4E package version
+        id: npm-version
+        uses: nyaa8/package-version@v1.0.3
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install Dependencies
+        run: npm ci
+        env:
+          NODE_OPTIONS: --max_old_space_size=4096
+
+      - name: Package VSIX
+        run: npx vsce package
```

Comment:
We have a 'package' npm script in the e4e package in our codebase, is it possible to use it instead?

In that case, we will reuse already existing packaging script, instead of duplicating packaging logic. Does it make sense?

Reply:
I'll try, I am not sure exactly how to use a different package.json than the one in root...

Reply:
Hmm, you can use `yarn workspace workspace_name script_name` syntax

Here is the [link](https://classic.yarnpkg.com/en/docs/cli/workspace/) to the doc

Reply:
Thank you!